api/lib/domain/usecases/create-assessment-result-for-completed-assessment.js

```diff
@@ -1,5 +1,6 @@
 const Assessment = require('../models/Assessment');
 const AssessmentResult = require('../models/AssessmentResult');
+const Promise = require('bluebird');
```

Comment:
pourquoi as-t-on besoin du Promise de bluebird ? Celui de js ne suffit pas ?

Reply:
hmm ok, c'est expliqué dans ce commit : "Use Promise.map instead of Promise.all", mais tu utilises le Promise de bluebird bien avant, c'est ce qui m'a perturbé.

Reply:
ouai. Quand je me suis rendu compte ça m'aurait fait faire un split de commit. J'ai pas eu le courage.

Reply:
Quand on utilise `bluebird` pour ses utilitaires je pense qu'il vaut mieux écrire :

```
const bluebird = require('bluebird');
…
await bluebird.map(…)
```

Plutôt que de masquer l'identifiant `Promise` de Node. On voit mieux quand on utilise une méthode non-standard, et on maîtrise l'impact — par exemple dans ce même fichier, l'appel à `Promise.all` dans `_saveResultAfterComputingError` est devenu un appel à `bluebird`…